aten/src/ATen/native/quantized/cpu/serialization_versions.h

```diff
@@ -0,0 +1,192 @@
+#pragma once
```

Comment:
I feel the name of the file is not very intuitive, why not just call it `serialization`?